MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -55,15 +55,29 @@ def store_issue_details(issue, repo, db):
 	issue_details['created_at'] = issue['created_at']
 	issue_details['updated_at'] = issue['updated_at']
 	issue_details['labels'] = issue['labels']
-	
+
 	issue_details['issue_content_path'] = issue_content_path
 	# issue_details['test_output_path'] = test_output_path
 	issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+	trss_jenkins_output_path = f'./data/GitHubData/JenkinsOutput/{issue_uid}'
 	#Store issue content at issue_content_path
 	store_on_fs(issue['body'], issue_content_path)
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	content_extracted = extract_jenkins_link_and_testname(issue['body'])
+	issue_quoted_jenkins_links = content_extracted[0]
+	issue_quoted_test_names = content_extracted[1]
+	if issue_quoted_jenkins_links != []:
+    		issue_details['issue_quoted_jenkins_links'] = issue_quoted_jenkins_links
+	if issue_quoted_test_names != []:
+    		issue_details['issue_quoted_test_names'] = issue_quoted_test_names
+
+	trss_jenkins_output = query_trss_for_jenkins_output(issue_quoted_jenkins_links, issue_quoted_test_names)
```

Comment:
`query_trss_for_jenkins_output` take only one link and one test name, so you need to loop your lists to query multiple times instead.

Reply:
Also, you need to check both lists exist before query.